src/Entity/Tradeability/Shipment.php

```diff
@@ -335,7 +331,7 @@ public function getProducts()
 
     /**
      * @param array $products
-     * @return QueryRequestShipment
```

Comment:
Is returning $this better than QueryRequestShipment? I don't know how this affects auto complete in IDE's. Could you help me what's the best choice?